### PR TITLE
chore: rename WsClientConenctionInfo and related params, methods, comments, etc

### PR DESCRIPTION
**Pull request by okjodom**
rename clarifies what the code is and what it's used for

### CODE REVIEW

File: devimint/src/bin/faucet.rs [hunk 1]
```diff
@@ -22,7 +22,7 @@ struct Cmd {
     #[clap(long, env = "FM_CLN_SOCKET")]
     cln_socket: String,
     #[clap(long, env = "FM_CONNECT_STRING")]
-    connect_string: Option<String>,
+    invite_string: Option<String>,
```
Comment (justinmoon) [inline devimint/src/bin/faucet.rs:25]: Can we always "invite code"? Having both "invite code" and "invite string" just invites confusion. Also, can you update the `FM_CONNECT_STRING` to `FM_INVITE_CODE`? Also, we have a `client-connect` file that save the invite code in devimint. Can we call that `invite-code`? Let's make sure we change it everywhere.

File: fedimint-cli/src/lib.rs [hunk 2]
```diff
@@ -361,7 +361,7 @@ enum Command {
 
     /// Join a federation using it's ConnectInfo
     JoinFederation {
-        connect: String,
+        invite: String,
```
Comment (justinmoon) [inline fedimint-cli/src/lib.rs:364]: Could we call this `invite_code`?

File: devimint/src/bin/faucet.rs (hunk 1 above)
Comment (wbobeirne) [reply devimint/src/bin/faucet.rs:25]: Fedimint only ever calls this `connect_string`, I don't see any results for `connect_code` or `invite_code`
Screenshot 2023-08-01 at 4 28 53 PM

**Comment by okjodom** [comment]
think I got em all

**Comment by justinmoon** [comment]
Seems like connection string encoding might be broken? Re-running CI in case but this sounds like it might not be a flake. https://github.com/fedimint/fedimint/actions/runs/5732408213/job/15542810909?pr=2882#step:9:245

File: devimint/src/bin/faucet.rs [hunk 3]
```diff
@@ -116,7 +116,7 @@ async fn main() -> anyhow::Result<()> {
         .route(
             "/connect-string",
```
Comment (okjodom) [inline devimint/src/bin/faucet.rs:117]: TODO: rename faucet route to `/invite-code`?

Comment (justinmoon) [reply devimint/src/bin/faucet.rs:117]: Rename it everywhere!

**Comment by justinmoon** [comment]
Would be good to fix that other one in follow-on pr.

**Review by justinmoon — APPROVED** [review]

**Pull request merged**